docs/en/04_Changelogs/4.0.0.md

```diff
@@ -278,6 +278,14 @@ E.g.
         'mysite/js/src/main.js',
         'mysite/js/src/functions.js'
     ]]);
+    
+### RestfulService
+
+* `RestfulService` has been removed. Use Guzzle instead. See Upgrading notes.
+
+### Oembed
+
+* `Oembed` has been removed from the framework in favor of [oscarotero/Embed](https://github.com/oscarotero/Embed)
```

Comment:
I'd make it clearer that oEmbed feature hasn't been dropped, just our code: "Our self-maintained Oembed implementation has been removed, in favour of introducing oscarotero/Embed as a dependency."